src/Containers/PictureCard.tsx

```diff
@@ -0,0 +1,142 @@
+    image: string;
+    tags?: TagProps[];
+    alt?: string;
+    height?: string;
+    width?: string;
+    borderradiustop?: string;
+    borderradiusbottom?: string;
+}
+
+export const PictureCard: React.FC<PictureCardProps> = ({
+    children,
+    image,
+    tags,
+    alt,
+    height,
+    width,
+    borderradiustop,
+    borderradiusbottom,
+    ...cardProps
+}): React.ReactElement => {
+    return (
+        <Card
+            width={width}
+            height={height}
+            borderradiustop={borderradiustop}
+            borderradiusbottom={borderradiusbottom}
+            {...cardProps}
+        >
+            <ImageWrapper>
+                <Image
+                    src={image}
+                    alt={alt}
+                    height={height}
+                    borderradiustop={borderradiustop}
+                    borderradiusbottom={borderradiusbottom}
+                />
+                {tags && (
+                    <Tags>
+                        {tags.map(
+                            ({ icon, text }): React.ReactElement => (
+                                <Tag>
+                                    {icon && <Icon as={icon} hasText={text} />}
+                                    {text}
+                                </Tag>
+                            ),
+                        )}
+                    </Tags>
+                )}
+            </ImageWrapper>
+            {children && <Content>{children}</Content>}
+        </Card>
+    );
+};
+
+interface CardProps {
+    width?: string;
+    height?: string;
+    borderradiustop?: string;
+    borderradiusbottom?: string;
+}
+
+const Card = styled(C)<CardProps>`
+    width: ${({ width }) => (width ? width : 'auto')};
+    height: ${({ height }) => (height ? height : 'auto')};
+    padding: 0px; //overriding styles
+    ${({ borderradiustop, borderradiusbottom }): string => `
+        border-top-left-radius:     ${borderradiustop};
+        border-top-right-radius:    ${borderradiustop};
+        border-bottom-left-radius:  ${borderradiusbottom};
+        border-bottom-right-radius: ${borderradiusbottom};
+    `}
+`;
+
+const ImageWrapper = styled.div`
+    position: relative;
+    ${flex('column')}
+`;
+
+const Image = styled.img<CardProps>`
+    width: 100%;
+    height: ${({ height }) => (height ? height : 'auto')};
+    ${({ borderradiustop, borderradiusbottom }): string => `
+        border-top-left-radius:     ${borderradiustop};
+        border-top-right-radius:    ${borderradiustop};
+        border-bottom-left-radius:  ${borderradiusbottom};
+        border-bottom-right-radius: ${borderradiusbottom};
+`}
+`;
+
+const Tags = styled.div`
+    ${flex('row', 'flex-start', 'stretch')}
+    ${position('absolute', '0 0 10px 10px', 'auto', 'auto', 0, 0)}
+    ${({ theme }): string => `
+        color:${theme.colors.text};
+    `}
+    font-size: 0.75rem;
+    font-weight: bold;
+`;
+
+const Tag = styled.span`
```

Comment:
Can we not have and/or use a tag component in the ui kit?

Reply:
Good Idea